test/functorch/test_vmap.py

```diff
@@ -3705,14 +3705,12 @@ def test_vmap_exhaustive(self, device, dtype, op):
         xfail('linalg.lu', ''),
         skip('linalg.ldl_solve', ''),
         skip('_softmax_backward_data'),
+        # AssertionError: Tensor-likes are not equal!
+        xfail('bitwise_left_shift', device_type='cpu'),
+        xfail('bitwise_right_shift', device_type='cpu'),
```

Comment:
It would be great if you can refer the issue #70904 here such that the next person reading this will have more context.

Reply:
Sure.